.github/workflows/build_release_candidate.yml

```diff
@@ -19,6 +19,9 @@ on:
       APACHE_PASSWORD:
         description: Your Apache password. Required if you want to stage artifacts into https://dist.apache.org/repos/dist/dev/beam/
         required: false
+      BEAM_SITE_TOKEN:
+        description:  Required if you want to create the beam-site docs PR.
```

Comment:
I assume this is a GitHub Actions PAT? Could we link to https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens and call out the necessary scopes (I assume it needs repo write; anything else?

Reply:
That's all my token has. That and "workflow" but I don't think that's necessary here.

If we add the pypi steps to an action we'll get additional use out of the token (same permissions), but that's a later change.